docs/source/user/getting-started.rst

```diff
@@ -4,32 +4,149 @@ Getting Started
+
+This will compile a debug version of the OpenDP shared library, placing it in the directory ``opendp/rust/target/debug``. (The specific name of the library file will vary depending on your platform.)
+
+Finally, you can create a local Python package that incorporates your new shared library. This is possible by using ``pip install`` with the ``-e`` option in the ``python`` subdirectory:
+
+.. prompt:: bash
+
+    cd ../python
+    pip install -e .
+
+At this point, you should be able use OpenDP as a locally installed package. You can test that things are working by running the OpenDP test suite, using ``pytest``:
+
+.. prompt:: bash
+
+    # Still in python subdirectory
+    pytest
+
+If everything has gone well, you'll see a bunch of output, then a line similar to this:
+
+    ================== 57 passed in 1.02s ==================
+
+This is just a quick overview of building OpenDP. If you're interested in porting OpenDP to a different platform, we'd be delighted to get your help; please :doc:`contact us <../contact/index>`!
 
 Hello, OpenDP!
 --------------
 
-Once you've installed OpenDP, you can write your first program. In the example below, we'll construct an identity ``Transformation``, then invoke it on a string.
+Once you've installed OpenDP, you can write your first program. In the example below, we'll construct a ``Transformation``, which is an OpenDP object that transforms data some way. In this case, the operation it performs is the identity transformation -- so no transformation at all! Then we'll apply that transformation to a string, getting back the original string.
 
 .. doctest::
 
     >>> from opendp.trans import make_identity
-    >>> from opendp.typing import SubstituteDistance
+    >>> from opendp.typing import SymmetricDistance
+
+    >>> identity = make_identity(M=SymmetricDistance, TA=str)
 
-    >>> identity = make_identity(M=SubstituteDistance, TA=str)
     >>> identity("Hello, world!")
     'Hello, world!'
+
+First, we import some types to have them in scope. ``make_identity`` is a constructor function, and ``SymmetricDistance`` is a type we need for disambiguation.
+
+Next we call ``make_identity()`` to construct an identity ``Transformation``. Because OpenDP is statically typed (even when called from dynamically typed languages like Python), we need to specify some type information. This is done by supplying some key-value arguments. ``M=SymmetricDistance`` says that we want the resulting ``Transformation`` to use the OpenDP type ``SymmetricDistance`` for its ``Metric``, and ``TA=str`` says that we want the ``Transformation`` to use the Python type ``str`` for its input and output type.
```

Comment:
Do you think this is a good place to explain the naming convention for type names? `TIA`, `TO`, `D`, etc?

Reply:
I think that's a little deep for a quick start, feels better later in user guide, maybe in program structure?